src/util/cors.ts

```diff
@@ -0,0 +1,24 @@
+import Cors from "cors";
+import { NextRequest, NextResponse } from "next/server";
+
+const corsConfig = Cors({
+  methods: ["GET", "POST", "PUT", "DELETE", "OPTIONS"],
+  origin: "*",
+});
+
+export function runMiddleware(
+  req: NextRequest,
+  res: NextResponse,
+  fn: Function,
+): Promise<void> {
```

Comment:
maybe add a check here that specifies that this is only allowed in sBTC testnet otherwise returns the res as is, if I understand this correctly in mainnet there will be apis that replace the proxy stuff